internal/minio/client/client.go

```diff
@@ -40,6 +41,10 @@ func Init() error {
 		minioClient = &MinioClient{
 			minioClient: c,
 		}
+
+		if initErr == nil {
+			utils.PrintLog("info", utils.LogLine{Message: "init", Category: "minio", Status: utils.SuccessStr})
```

Comment:
this is a success, why should it be an error level? not sure to understand your point. the print is here to because it has to be displayed only once, and to so, it has to be in the Once.Do block

Reply:
completely my mistake, removing all comments from the same kind.